src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -1,11 +1,12 @@
+/* eslint-disable @stylistic/brace-style */
```

Comment:
I'm not in favor of disabling this rule here as it helps having a consistent coding style.
Please try to restructure the comments / code without moving the braces.

Reply:
Removed.